jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1082,6 +1083,8 @@ public void endpointMessageTransportConnected(@NotNull AbstractEndpoint abstract
             {
                 endpoint.sendMessage(new DominantSpeakerMessage(recentSpeakers, speechActivity.isInSilence()));
             }
+
+            abstractEndpoint.onMessageTransportConnect();
```

Comment:
We already have `Endpoint.endpointMessageTransportConnected`. I don't see you implement this in `Relay`, so I think you can just merge your code into it.

Reply:
thanks, fixed.